src/site/stages/build/drupal/metalsmith-drupal.js

```diff
@@ -206,7 +206,7 @@ function getDrupalContent(buildOptions) {
       }
       drupalData = convertDrupalFilesToLocal(drupalData, files, buildOptions);
 
-      loadCachedDrupalFiles(buildOptions, files);
+      await loadCachedDrupalFiles(buildOptions, files);
```

Comment:
Discovered we weren't `await`ing this `async` function. Not sure what the impact is here, honestly.

@ncksllvn?